### PR TITLE
feat: extend order type

### PR DESCRIPTION
**Pull request by chybisov**

### CODE REVIEW

File: src/api.ts [hunk 1]
```diff
@@ -9,7 +9,8 @@ import {
 import { Bridge } from './bridges'
 import { Exchange, ExchangeAggregator } from './exchanges'
 
-export type Order = 'RECOMMENDED' | 'FASTEST' | 'CHEAPEST' | 'SAFEST'
+export const _Order = ['RECOMMENDED', 'FASTEST', 'CHEAPEST', 'SAFEST'] as const
+export type Order = typeof _Order[number]
```
Comment (mathiasmoeller) [inline src/api.ts:13]: Why do you need this? Also `_` usually indicates a private property so exporting it feels a bit weird I think?

Comment (chybisov) [reply src/api.ts:13]: We need this for route priority selection in the widget. I made it like this to follow other unions naming in this file. We can rename it if needed. 🙂

Comment (chybisov) [reply src/api.ts:13]: Orders will be fine.

Comment (chybisov) [reply src/api.ts:13]: Yes, already been published. :)

**Pull request merged**